.github/workflows/workflow-validate-website-content.yml

```
@@ -5,7 +5,10 @@ on:
       site_git_ref:
         description: 'git ref, a SHA commit or branch or tag, for the published HTML content'
         required: true
-        default: 'nist-pages'
+      site_git_ref:
```

actionlint (1.7.4)

key "site_git_ref" is duplicated in "inputs" section. previously defined at line:5,col:7. note that this key is case insensitive

(syntax-check)